test/testcafe/spec/ChallengeGoogleAuthenticatorOtp_spec.js

```diff
@@ -79,10 +86,17 @@ test
   });
 
 test
-  .requestHooks(invalidOTPMock)('challenge google authenticator with invalid TOTP', async t => {
+  .requestHooks(invalidPasscodeMock)('challenge google authenticator with invalid passcode', async t => {
+    const challengeGoogleAuthenticatorPageObject = await setup(t);
+    await challengeGoogleAuthenticatorPageObject.verifyFactor('credentials.passcode', '123');
+    await challengeGoogleAuthenticatorPageObject.clickNextButton();
+    await t.expect(challengeGoogleAuthenticatorPageObject.getAnswerInlineError()).contains('Your code doesn\'t match our records. Please try again.');
```

Comment:
A better way to test this is modify message text in actual error response and make sure the text in our properties files is picked up right now they both are same and we cannot tel for sure if it got pulled form login.prop 

Same for the other string as well. 

You can probably update the message here 

https://github.com/okta/okta-signin-widget/pull/1849/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.properties&hide-deleted-files=true#diff-5ee3afaf6d4301f6a9827184ea586a774e38b090bf118208058f94b2dafef0b3R34